plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -1828,6 +1836,7 @@ protected void testTargetMaxFileSize(int expectedTableWriters)
         Session session = Session.builder(getSession())
                 .setSystemProperty("task_writer_count", "1")
                 .setSystemProperty("scale_writers", "false")
+                .setSystemProperty("task_scale_writers_enabled", "false")
```

Comment:
Can we enable it? If no then please add a comment? Is it possible to extract just tests that count files?

Reply:
> Is it possible to extract just tests that count files?

WDYM?

Reply:
Added a comment

Reply:
Not all, only `testTargetMaxFileSize` and scale writers ones